webpack/assets/javascripts/react_app/components/HostDetails/Status/StatusActions.js

```diff
@@ -1,21 +1,28 @@
 import { APIActions } from '../../../redux/API';
-import { HOST_STATUSES_KEY } from './Constants';
+import { HOST_STATUSES_KEY, CLEAR_STATUS_KEY } from './Constants';
 import { foremanUrl } from '../../../common/helpers';
 
+const getStatuses = hostName => dispatch => {
+  const url = foremanUrl(`/hosts/${hostName}/statuses`);
+  dispatch(
+    APIActions.get({
+      url,
+      key: HOST_STATUSES_KEY,
+    })
+  );
+};
+
 export const forgetStatus = (hostName, { label, id }) => dispatch => {
   const successToast = () => `Status ${label} has been removed`;
   const errorToast = ({ message }) => message;
   const url = foremanUrl(`/hosts/${hostName}/forget_status?status=${id}`);
   dispatch(
     APIActions.post({
       url: foremanUrl(url),
-      key: HOST_STATUSES_KEY,
+      key: CLEAR_STATUS_KEY,
       successToast,
       errorToast,
-      updateData: prevState => ({
-        ...prevState,
-        statuses: prevState.statuses.filter(status => status.id !== id),
-      }),
+      handleSuccess: () => dispatch(getStatuses(hostName)),
```

Comment:
After a successful status clearing, we need to pull the refreshed global status from the server so the user doesn't need  to refresh the entire page